### DISCUSSION TITLE
How to check if the update was successful

### DISCUSSION THREAD

**Discussion by Laudings** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#376] [Q&A]
Hello,

i updated from V3.3.1 to 3.4 via OTA. Report says successful. 
In 3.3.1 i modified the web interface a bit. Now after the update to V3.4 i thougt the web interface is now also stock. but it isn't. It is still my modified version. 
So now my question: Is it possible to check which version is on the pcb? Because i am not sure if the update was truely succesful.
Thanks!

**Comment by visualapproach** [comment]
Hi, the files in the /data folder (All the html) must also be uploaded. Either by uploading as an image (destroys all previous save files) or file by file through the webpage "layzspa/upload.html". FW date is shown in the bottom of the index.html page. (After a certain version)

**Comment by Laudings** [comment]
OK thank you! I want to unterstand. Is this neccessary because the OTA update? Because it wasn't neccessary by the first flashing via USB.
Do i get some problems if i don't update the data folder in the future? Thanks!

**Reply by visualapproach** [reply]
You certainly did and I really made what I could in the build instructions to emphasize that.
Sketch/program goes to one area of the flash memory and data files to another. You have to upload both to be sure. Sometimes there is no change in the data files so then you will not need to upload again ofc. May be hard to know but I usually comment every change in the release notes. Not super easy to find but click on the version/release tag and show all history.

**Reply by Laudings** [reply]
OK thank you again for your explanation.

**Reply by visualapproach** [reply]
No problem 😉
